firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionTesterApiClient.java

```diff
@@ -14,13 +14,11 @@
 
```

Comment:
Can we log the id of newRelease in this file? Maybe around line 118?

Reply:
I have logged the apk hash for the release not sure if that is what you meant by id.